DUnitX.TestFramework.pas

```diff
@@ -584,22 +602,6 @@   TDUnitX = class
     function GetReport: string;
   end;
 
-
-  ETestFrameworkException = class(Exception);
```

Comment:
Happy with that. Talking with Vincent we are fine with the change in public interface as no one should really be using this "feature/work around".